yowsup/layers/protocol_media/protocolentities/message_media_downloadable.py

```diff
@@ -20,10 +20,10 @@ class DownloadableMediaMessageProtocolEntity(MediaMessageProtocolEntity):
     '''
     def __init__(self, mediaType,
             mimeType, fileHash, url, ip, size, fileName, 
-            _id = None, _from = None, to = None, notify = None, timestamp = None, participant = None,
-            preview = None, offline = None, retry = None):
+            _id = None, _from = None, to = None, notify = None, timestamp = None, 
+            participant = None, offline = None, retry = None):
 
```

Comment:
I moved the ```preview``` attribute to each ```video```/```picture```/```location``` class because ```audio``` and ```vcard``` don't have previews and thus it isn't a common attribute to all kinds of ```DownloadableMediaMessageEntity```.

If you reckon that it should remain as a common attribute, please tell me and I'll move it back to ```DownloadableMediaMessageEntity```.